apps/pwa/src/pages/settings/providers/index.tsx

```
@@ -622,160 +610,191 @@ export default function ModelPage({ className }: { className?: string }) {
+                    }),
+                  )}
+                  value={openrouterProviderSort?.toString()}
+                  onValueChange={(value) => {
+                    setOpenrouterProviderSort(value as OpenrouterProviderSort);
+                  }}
+                />
+                <div className="flex items-center gap-1.5 text-xs text-fg-muted">
+                  <Info size={12} />
+                  <a
+                    href="https://openrouter.ai/docs/features/provider-routing#price-based-load-balancing-default-strategy"
+                    target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)